CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
   * For the EC encryptor, the encoded public key length is 91
 
 ### Additions and Improvements
+
+- Reduce the number of runtime exceptions (SecurityModuleException) and unnecessary executions during ECIES handshake [#4508](https://github.com/hyperledger/besu/pull/4508)
```

Comment:
I would also explain that we are reducing these exceptions try EIP-8 formatted messages first